swap_meet/item.py

```diff
@@ -1,2 +1,40 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        if id == None:
```

Comment:
Remember, if we are checking if something is `None`, we should do `if id is None`